rcl/src/rcl/rcl.c

```diff
@@ -118,10 +119,16 @@ rcl_init(int argc, char const * const * argv, rcl_allocator_t allocator)
     goto fail;
   }
 
-  // Update the default log level if specified in arguments.
-  if (global_args->impl->log_level >= 0) {
-    rcutils_logging_set_default_logger_level(global_args->impl->log_level);
+  fail_ret = rcutils_logging_configure(global_args->impl->log_level,
+                                      global_args->impl->external_log_config_file,
+                                      !global_args->impl->log_stdout_disabled,
+                                      !global_args->impl->log_rosout_disabled,
+                                      !global_args->impl->log_ext_lib_disabled);
```

Comment:
nitpick: This doesn't follow our style I think, but again, you can use our linters to see the diff it thinks should be made or use the `--reformat` option to have it do it for you (in the case of `ament_uncrustify`) at least.

The linters are also run as automated tests (via CTest).

Reply:
Fixed with ament_uncrustify.